pandas/_libs/tslibs/nattype.pyx

```diff
@@ -587,3 +588,28 @@ cdef inline bint checknull_with_nat(object val):
     """ utility to check if a value is a nat or not """
     return val is None or (
         PyFloat_Check(val) and val != val) or val is NaT
+
+
+cdef inline bint is_null_datetimelike(object val):
+    """
+    Determine if we have a null for a timedelta/datetime (or integer versions)
+
+    Parameters
+    ----------
+    val : object
+
+    Returns
+    -------
+    null_datetimelike : bool
+    """
+    if util._checknull(val):
+        return True
+    elif val is NaT:
+        return True
+    elif util.is_timedelta64_object(val):
```

Comment:
why import this from  `util`? Didn't you move it to `tslibs.missing` ?

Reply:
Not sure I understand the question.  This PR is moving this function from `_libs.missing` to `tslibs.nattype` because doing so simplifies the dependency structure.

Reply:
and by moving the function should not change the import?

Reply:
these util functions of basically c versions of isinstance checks, so this is ok.